lib/t/matches.ex

```diff
@@ -407,7 +360,6 @@ defmodule T.Matches do
       end
     end)
     |> delete_likes()
-    |> insert_expired_match()
```

Comment:
This seems more like unmatch. Should this function do something more than unmatching or should that operation be named differently?

Reply:
@ruslandoga you are right: this function should also "bump" seen status (so that expired mates do not see each other for seen_prune period)
thanks for noticing 👍